### PR TITLE
Make a folder for examples. Close #477.

### LINKED ISSUE #477: Make a new folder for examples
Files under `test/` should be simple (unless such complexity is necessary for testing some functionality). Univalence, pi1(S1), semi-simplicial types, etc do not belong there. @cangiuli

### PR DESCRIPTION
**Pull request by favonia**

### CODE REVIEW

File: script/test-no-build.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@
 REDPRL=./bin/redprl
 PROBLEM=0
 
-# Ensure that the examples file and all suceeding tests succeed
-for f in test/examples.prl test/success/*.prl ; do
+# Ensure that all suceeding tests and examples type-check
```
Comment (cangiuli) [inline script/test-no-build.sh]: succeeding

**Comment by favonia** [comment]
@cangiuli Fixed. The line was arbitrarily drawn with 1.38-2.42% of my brain power.

**Comment by jonsterling** [comment]
looks good

**Comment by cangiuli** [comment]
Great, feel free to merge.

**Pull request merged**